lib/new_relic/agent/instrumentation/ruby_openai/instrumentation.rb

```diff
@@ -101,8 +102,7 @@ def create_chat_completion_messages(parameters, summary_id)
           role: message[:role] || message['role'],
           sequence: index,
           completion_id: summary_id,
-          vendor: VENDOR,
-          is_response: true
```

Comment:
This is an unrelated fix. This first group of messages are not from a response. They're from a request. The `is_response` should only be attached to the event if it has a truthy value.

Reply:
nice